sdk/log/logtest/builder.go

```diff
@@ -0,0 +1,131 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package logtest // import "go.opentelemetry.io/otel/sdk/log/logtest"
+
+import (
+	"context"
+	"slices"
+	"time"
+
+	"go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
+	sdklog "go.opentelemetry.io/otel/sdk/log"
+	"go.opentelemetry.io/otel/sdk/resource"
+)
+
+// TODO: comment.
+type Builder struct {
```

Comment:
How about `RecordBuilder`?

Reply:
This was my initial idea 3c29498

However I think that it is not needed. We also do not have RecordProcessor nor RecordExporter

Reply:
Even though it's unlikely there will ever be any builder other than for record, calling `logtest.Builder{}` is a bit non-expressive, as it doesn't say what we're going to build.
So I'd go with @XSAM in favor of `RecordBuilder`.

Reply:
OK. I will revert then 😉